testprojects/src/thrift/org/pantsbuild/thrift_exports/BUILD

```diff
@@ -0,0 +1,61 @@
+java_thrift_library(
+  name='A',
+  sources=[
+    'A.thrift',
+  ],
+  language='scala',
+  compiler='scrooge',
+)
+
+java_thrift_library(
+  name='B1',
+  sources=[
+    'B.thrift',
+  ],
+  dependencies=[
+    ':A',
+  ],
+  exports=[
+    ':A',
+  ],
+  language='scala',
+  compiler='scrooge',
+)
+
+java_thrift_library(
+  name='B2',
+  sources=[
+    'B.thrift',
+  ],
+  dependencies=[
+    ':A',
+  ],
+  language='scala',
+  compiler='scrooge',
+)
+
+java_thrift_library(
+  name='C1',
```

Comment:
It'd be nice if the names for these targets included something about their config. C-with-exporting-dependency or something would be clearer when they are referenced in tests.